REPOSITORY-GUIDELINES.md

```diff
@@ -0,0 +1,131 @@
+* [Lifecycle](#lifecycle)
+  * [Addition](#addition)
+    * [Adding ```Core``` repositories](#adding-core-repositories)
+  * [Change of scope or status](#change-of-scope-or-status)
+    * [Archival](#archival)
+* [Credit](#credit)
+
+## Scope
+
+Each repository within the Cilium GitHub organization is assigned one of four scopes: ```Core```, ```Ecosystem```, ```Infra```, or ```Special```. This assigned scope helps to convey each repository's purpose and responsibility in the overall functioning of the project.
+
+### ```Core```
+
+Core repositories are at the heart of the Cilium project, and represent its most essential functioning. They are the most active in terms of both contributions and use cases, and represent the repositories that adopters will typically interact with most often.
+
+Notable repositories of this type include [Cilium](https://github.com/cilium/cilium), [eBPF](https://github.com/cilium/ebpf), [Hubble](https://github.com/cilium/hubble), and [Tetragon](https://github.com/cilium/tetragon).
+
+You can find the list of all ```Core``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#core-scope).
+
+### ```Ecosystem```
+
+Ecosystem repositories are extensions of the core project, providing optional functionality that adopters may find useful. While not considered essential, they play a pivotal role of addressing a range of use cases and needs for Cilium adoptors.
+
+Examples of repositories of this type include useful Go libraries like [Fake](https://github.com/cilium/fake), or repositories designed to integrate Cilium with other projects like [Proxy](https://github.com/cilium/proxy).
+
+You can find the list of all ```Ecosystem``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#ecosystem-scope).
+
+### ```Infra```
+
+Infrastructure repositories are those that are required to keep the Cilium project running smoothly, and are not designed to be used by adopters.
+
+Examples of repositories of this type include development and testing tools like [little-vm-helper](https://github.com/cilium/little-vm-helper), or those helping with the overall contribution process like [github-actions](https://github.com/cilium/github-actions).
+
+You can find the list of all ```Infra``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#infra-scope).
+
+### ```Special```
+
+Special repositories are those where assigning a status is not appropriate. They are generally not designed to be used by adoptors.
+
+They include the [Community](https://github.com/cilium/community) and [.github](https://github.com/cilium/.github) repository, template repositories like [design-cfps](https://github.com/cilium/design-cfps), as well as all forks.
+
+You can find the list of all ```Special``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#special-scope).
+
+## Status
```

Comment:
I've discussed this a bit with Joe and we think that for now, we shouldn't have status except for deprecated. It adds more process without much benefit for end users and seems to add more "guarantees" about the software that the license doesn't actually agree with.

For core repos, end users can either choose to adopt or not depending on their needs while the other ones are just up to project members to choose and it is up to them to also keep the repos/projects up to date with their needs.

Reply:
Ok, I can take the status parts out. That does make it a lot simpler